org.lflang/src/org/lflang/generator/ts/TSConstructorGenerator.kt

```diff
@@ -105,6 +107,17 @@ class TSConstructorGenerator (
         return connectionInstantiations.joinToString("\n")
     }
 
+    // Generate code for setting target configurations.
+    private fun generateTargetConfigurations(): String {
+        val targetConfigurations = LinkedList<String>()
+        if ((reactor.isMain || reactor.isFederated) &&
+            targetConfig.coordinationOptions.advance_message_interval != null) {
+            targetConfigurations.add(
+                "this.setAdvanceMessageInterval(${timeInTargetLanguage(targetConfig.coordinationOptions.advance_message_interval)})")
+        }
```

Comment:
I think there should be a warning here if advance_message_interval is not set by the user. At least this is how this scenario is handled in C and Python.

Reply:
Thank you for the suggestion! I added a warning here: https://github.com/lf-lang/lingua-franca/pull/1293/commits/1c6a598202f12cbf1b3ab902f5ad872a4d39f3af